src/content/docs/new-relic-solutions/observability-maturity/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Observability Maturity
+type: landingPage
+tags:
+  - Observability maturity
+  - Uptime, performance, and reliability
+  - Operational Efficency
+  - Customer Experience
+  - Innovation and Growth
+  - Implementation guide
+metaDescription: Observability Maturity landing page
+redirects:
+  - /docs/solutions-best-practices-landing-page
+---
+
+<LandingPageHero>
+
+ <HeroContent>
+   New Relic Observability Maturity Architecture (OMA) is now open-source. Observability Maturity Architecture is the practice of discovering, curating, and documenting scalable and repeatable best practices in the Observability Industry. This move will allow all to contribute to the standards of Observability Maturity Architecture.
```

Comment:
Some edit suggestions: 

- remove "now" from the first sentence so it's got longevity.
- lower case "observability industry"